### ISSUE TITLE
Does not work froperly with files like .xlsx or images (extra bytes are present in buffer)

### ISSUE THREAD

**Issue by yuriygavriluk** [dmitriy-nz/nestjs-form-data#13]
Upload excel file and save it to disk or log request
@Module({
  imports: [NestjsFormDataModule.config({ storage: FileSystemStoredFile,
    fileSystemStoragePath: 'nestjs-fd'
  })],
  providers: [
  ],
  exports: [NestjsFormDataModule],
})

run request thewre excal or image is form data and log request

Problem if you go to 'nestjs-fd' folder the file will be corrupted
if you check log file size is bigger then actual files that was sent and extra bytes are present

**Comment by yuriygavriluk** [comment]
Yes I think this is only related to AWS Api gateway so not a topic for this library 
Ticket can be closed

**Comment by dmitriy-nz** [comment]
It's good that the problem was found
